pool/rate_limiting_test.go

```diff
@@ -154,3 +156,23 @@ func TestEWMARateLimitingWorkerHandlesPanicingJobs(t *testing.T) {
 	}
 	assert.NotPanics(func() { p.worker(ctx, jobsChanWithPanicingJobs(ctx, 10)) })
 }
+
+func TestMultipleWorkers(t *testing.T) {
+	assert := assert.New(t) // nolint
+	for workers := 1; workers <= 100; workers++ {
+		ema := ewmaRateLimiting{
+			period: time.Minute,
+			target: 60,
+			size:   workers,
+			queue:  nil,
+			ewma:   ewma.NewMovingAverage(),
+		}
+		for i := 0; i < 100; i++ {
+			next := ema.getNextTime(time.Millisecond)
+
+			if !assert.True(next > 750*time.Millisecond) || !assert.True(next < time.Second) {
```

Comment:
The sleep should increase as the number of workers increases. I believe my solution was correct.